filebeat/_meta/config/filebeat.global.reference.yml.tmpl

```diff
@@ -4,14 +4,14 @@
 # data path.
 #filebeat.registry.path: ${path.data}/registry
 
-# The permissions mask to apply on registry data, and meta files. The default
+# The permissions mask to apply on registry data and meta files. The default
 # value is 0600.  Must be a valid Unix-style file permissions mask expressed in
 # octal notation.  This option is not supported on Windows.
 #filebeat.registry.file_permissions: 0600
 
-# The timeout value that controls when registry entries are written to disk
+# The timeout value that controls when registry entries are written to the disk
 # (flushed). When an unwritten update exceeds this value, it triggers a write
-# to disk. When flush is set to 0s, the registry is written to disk after each
+# to disk. When the flush is set to 0s, the registry is written to disk after each
```

Comment:
`flush` in this context is being used as the variable name. Making it "the flush" sounds worse IMO. I would either leave this as is or change it to "When the value is set to 0s".

Reply:
Okay, I'll do it.

Reply:
Done!